modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -510,15 +510,51 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
     using ParamType = std::pair<InferenceEngine::TensorDesc,
```

Comment:
Not for this changes but:
`auto ie_core = cv::gimpl::ie::wrap::getCore();` - Is unused variable?

Reply:
you right, it seems is unused